client/src/components/QandA/AddQuestion.jsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+class AddQuestion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      questionEntry: '',
+      nicknameEntry: '',
+      emailEntry: '',
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(e) {
```

Comment:
I saw your comment on the PR description that this function is not working correctly. 
I suggest adding a comment here too (this is just a suggestion to make it easier to know when you merge this into main).

Reply:
There is a comment just below on line 17 specifying how it needs to be fixed which will let me know what to do in a later PR - do you mean I should add an additional comment to let others know that this isn't yet implemented? I thought mentioning this in my PR would suffice since nobody else will be using this.

Reply:
Done.